.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -88,6 +94,58 @@ steps:
         notify:
           - github_commit_status:
               context: "agentbeat: Packaging"
+      - label: ":linux: Agentbeat packaging linux/amd64 FIPS"
+        key: "agentbeat-package-linux-x86-fips"
+        depends_on:
+          - mandatory-win-2022-system-tests
+        env:
+          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
+          SNAPSHOT: true
+          FIPS: "true"
+        command: |
+          .buildkite/scripts/packaging/packaging.sh x-pack/agentbeat
+        artifact_paths:
+          - x-pack/agentbeat/build/distributions/**/*
+          - "x-pack/agentbeat/build/*.xml"
+          - "x-pack/agentbeat/build/*.json"
+        retry:
+          automatic:
+            - limit: 2
+        timeout_in_minutes: 60
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        notify:
+          - github_commit_status:
+              context: "agentbeat: Packaging linux/amd64 FIPS"
+      - label: ":linux: Agentbeat packaging linux/arm FIPS"
```

Comment:
```suggestion
      - label: ":linux: Agentbeat packaging linux/arm64 FIPS"
```

Reply:
Done in f35490f4f2b69447ed85f6716817e581bee82e95